docs-ref-conceptual/includes/cli-install-linux-dnf.md

```diff
@@ -35,21 +35,25 @@ for the Azure CLI. This package has been tested with RHEL 7.7, RHEL 8, Fedora 24
    gpgkey=https://packages.microsoft.com/keys/microsoft.asc" | sudo tee /etc/yum.repos.d/azure-cli.repo
    ```
 
-3. Install with the `yum install` command.
+3. Install with the `dnf install` command.
 
    ```bash
-   sudo yum install azure-cli
+   sudo dnf install azure-cli
    ```
-
+ 
 Run the Azure CLI with the `az` command. To sign in, use [az login](/cli/azure/reference-index#az-login) command.
+ 
+> [!NOTE]
+>
+> Use `yum` package manager if you are using Linux systems that don't support `dnf` package manager.
```

Comment:
Can we move up this note under `Overview`, probably below the `Important` notice?

Reply:
Sure. I will move it under the overview section.